guide/contributing_guide.md

```diff
@@ -1,4 +1,23 @@
-# Coding style
+# Contributing Guide
+
+## General contribution info
+Any contributions to Membrane Framework are welcome. If you would like contribute, but you're not sure how to start or have some questions, don't hesitate to contact us via our [Discord](https://discord.gg/nwnfVSY).
+<br>
+For inquiries related to our commercial services or for other contact information, please visit [Membrane website](https://membrane.stream/contact) for more information.
+<br>
+When contributing to existing repo:
+- fork it
+- apply change on some branch with a meaningful name
+- create a PR from fork to our repo
+- await feedback from someone from the team
+- after passing the review, the PR will be merged
+<br>
+
+If you wish to create a new plugin you can give us a shout and if it's something we want as a part of our ecosystem we'll create a repo for you, guide you on your work and maintain the plugin in future. 
+<br>
+However, if feel confident enough to maintain it on your own, you can, of course, create your own repo and hex package - we only ask you to follow the naming conventions of the framework package and modules. In that case, don't forget to let us know about your work so we could include it on the list of available plugins.
```

Comment:
Those paragraphs feel odd to me. Maybe double `<br>` or no `<br>` at all?

Reply:
Fixed